src/views/ModelPlan/TaskList/ITSolutions/index.tsx

```diff
@@ -22,6 +23,20 @@ const ITSolutions = () => {
               <ITSolutionsHome />
             </Route>
 
+            <Route
+              path="/models/:modelID/task-list/it-solutions/add-an-operational-need"
+              exact
+            >
+              <AddOrUpdateOperationalNeed />
+            </Route>
+
+            <Route
+              path="/models/:modelID/task-list/it-solutions/update-need/:operationalNeedID?"
+              exact
+            >
+              <AddOrUpdateOperationalNeed />
+            </Route>
+
```

Comment:
Just a code alternative to this style for simplicity would be:

```
<Route
    path="/models/:modelID/task-list/it-solutions/custom-operational-need/:operationalNeedID?
    exact
  >
    <AddOrUpdateOperationalNeed />
  </Route>
```

This route doesn't really need to be duplicated, can be condensed to one route with an optional parameter if updating.